### DISCUSSION TITLE
🔗 Back to the top button (Duplicate of #3184)

### DISCUSSION THREAD

**Discussion by nickdgreen** [elementor/elementor#21914] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

A request for a back-to-the-top widget was logged on Jan 15, 2018 #3184 but there has been no progress on delivering the feature. Lots of flashy widgets added but a simple function for the visitor's ease of navigation has been omitted. I have a personal solution for this but it is manual to implement and should be something that can be added and styled within Elementor.

### Describe the solution you'd like

There are two methods by which this could be implemented:
Option 1 - a back-to-the-top widget that could be placed in the page footer. This would be essentially a button that could contain an icon (chosen in the "normal" method from the icon library), text, or both. The button itself would be provided with the usual setting options text colour, background colour, background transparency etc. For the button to function it would require a supporting javascript or jquery to control the button's appearance on the page and fixing to the bottom of the page. Using a widget the designer could choose whether the button appeared to the left, right, or centred on the page by placing it in the appropriate place. The disadvantage of a widget is that it would not appear on any page on which the footer had been excluded, for example, a landing page.
Option 2 - a back-to-the-top button as a selectable item within the Site Setting. Initially, this would be a Back To The Top Button Yes/No switch. If Yes were selected then this would open up the configuration options for the button; these being the same as in option 1 above as regards size, colour etc. with an additional position selector for left, right or centred. To be this would be a better option as the feature could be enabled regardless of whether a footer is present on a page and it would also not interfere with the design of the footer. 

### Describe alternatives you've considered

The alternative that I use is a function in a Hello Child Theme a js file and manual CSS styling that is outside the skill set of many of the users of Elementor.

**Comment by nicholaszein** [comment]
Hi!

This is already technically possible. All you need to do is to add a CSS ID to the first Container in the Header template, and then link to that template in a Button widget with it's position set to absolute. You already have all the tools for achieving this.

Please, do not create duplicate Feature Requests though. This only makes it more difficult to verify the popularity of one of the features.

I'm closing this one since there is already a request for this feature.

Thank you for contributing! 🙏
